vspace/src/main/java/edu/asu/diging/vspace/web/staff/DeleteBiblioBlockController.java

```diff
@@ -0,0 +1,80 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.diging.vspace.core.exception.BlockDoesNotExistException;
+import edu.asu.diging.vspace.core.model.IModule;
+import edu.asu.diging.vspace.core.model.IReference;
+import edu.asu.diging.vspace.core.model.ISlide;
+import edu.asu.diging.vspace.core.services.IContentBlockManager;
+import edu.asu.diging.vspace.core.services.IModuleManager;
+import edu.asu.diging.vspace.core.services.IReferenceManager;
+import edu.asu.diging.vspace.core.services.ISlideManager;
+
+@Controller
+public class DeleteBiblioBlockController {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    @Autowired
+    private IContentBlockManager contentBlockManager;
+    
+    @Autowired
+    private IModuleManager moduleManager;
+    
+    @Autowired
+    private ISlideManager slideManager;
+    
+    @Autowired
+    private IReferenceManager referenceManager;
+
+    @RequestMapping(value = "/staff/module/{moduleId}/slide/{id}/biblio/{blockId}", method = RequestMethod.DELETE)
+    public ResponseEntity<String> deleteBiblioBlock(@PathVariable("moduleId") String moduleId, @PathVariable("id") String slideId, 
+            @PathVariable("blockId") String blockId) throws IOException {
+        
+        ISlide slide = slideManager.getSlide(slideId);
+        IModule module = moduleManager.getModule(moduleId);
+        // If the slide Id and module Id is not found, show message on screen.
+        if(slide==null) {
+            logger.warn("Slide Id does not exist, bad request.");
+            return new ResponseEntity<String>(HttpStatus.BAD_REQUEST);
+        }
+        
+        if(module==null) {
+            logger.warn("Module Id does not exist, bad request.");
+            return new ResponseEntity<String>(HttpStatus.BAD_REQUEST);
+        }
+        
+        List<IReference> refList = referenceManager.getReferencesForBiblio(blockId);
+        
+        if(refList.size()>0) {
+            try {
+                referenceManager.deleteRefences(refList, blockId);
```

Comment:
for now I would not delete the references, just the block. they are two different entities that we want to manage independently of each other. maybe later we'll add an option to remove references with the block, but for now let's just remove the block.